hoomd/hpmc/pytest/test_faceted_ellipsoid_union.py

```diff
@@ -0,0 +1,591 @@
+import hoomd
```

Comment:
This whole file could benefit from more uses of for loops. If you are going to leave the args as module level variables you can access them programmatically with `globals()[key]`. I really recommend pytest fixtures, however.